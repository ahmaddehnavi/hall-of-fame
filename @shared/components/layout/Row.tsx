import * as React from 'react';
import {View, StyleSheet, ViewProps} from 'react-native';

/**
 * use Row or Col to make jsx more readable
 */
export function Row(props: ViewProps & { children?: React.ReactNode }) {
    return (
        <View
            {...props}
            style={[styles.row, props.style]}/>
    )
}

const styles = StyleSheet.create({
    row: {flexDirection: 'row'}
});