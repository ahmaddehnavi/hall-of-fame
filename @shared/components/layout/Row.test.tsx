import React from 'react';
import 'react-native';
// Note: test renderer must be required after react-native.
import renderer from 'react-test-renderer';
import {Row} from './Row';

describe('Row Tests', () => {

    it('should match snapshot', () => {
        let tree = renderer.create(
            <Row

            />
        );
        expect(tree).toMatchSnapshot();
    });

});
