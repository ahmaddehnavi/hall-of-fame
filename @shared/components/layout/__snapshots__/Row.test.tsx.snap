// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Row Tests should match snapshot 1`] = `
<View
  style={
    Array [
      Object {
        "flexDirection": "row",
      },
      undefined,
    ]
  }
/>
`;
